homeassistant/components/zwave_js/__init__.py

```diff
@@ -443,6 +445,14 @@ def async_on_notification(
                     ATTR_ACKNOWLEDGED_FRAMES: notification.acknowledged_frames,
                 }
             )
+        elif isinstance(notification, MultilevelSwitchNotification):
+            event_data.update(
+                {
+                    ATTR_COMMAND_CLASS_NAME: "Multi Level Switch",
```

Comment:
zwave-js uses `Multilevel Switch` as the command class name. @MartinHjelmare thoughts on whether the extra space should be included or not?

Reply:
I'd vote for `Multilevel Switch`, although this appears to be done the same way as `Power Level`, which is really the `Powerlevel` command class. Too late to change the latter (double anyone's using it...)?

Would it be better if command class names were set in a const somewhere in the python lib?

Reply:
We made an explicit decision to split power level into two words, I'll have to find the discussion later. Anyway, that's the only case where we've broken convention from Z-Wave JS. Does the zwave spec consider it to be one word? If so maybe we *should* change it

Reply:
I agree it should be Multilevel Switch, I think I was trying to keep it consistent with Power Level, elsewhere Powerlevel is shown as a single word.

Reply:
We changed the name of power level in our client api methods.

https://github.com/home-assistant-libs/zwave-js-server-python/pull/328#issuecomment-985657452

It was not the intention to change the name of the Command Class. The command class has a name and we can't change that.

Reply:
OK I can revert the command class name here, I doubt anyone's using this notification type but we can list it as a breaking change. I still don't see where the line is drawn between the models in the lib and the command class since the commands are all related to this particular command class I believe